godotenv.go

```diff
@@ -209,7 +246,7 @@ func readFile(filename string) (envMap map[string]string, err error) {
 	return Parse(file)
 }
 
-func parseLine(line string, envMap map[string]string) (key string, value string, err error) {
+func parseLine(line string, envMap map[string]string, isMultiline bool) (key string, value string, multilineValue bool, err error) {
```

Comment:
I'd appreciate if you could find a way to restructure this to not rely on a boolean flag as I'm in Fowler's camp on the topic https://www.martinfowler.com/bliki/FlagArgument.html